frameworks/engineering/mobile.md

```diff
@@ -186,6 +187,7 @@ topics:
           - "Appropriately uses common functional reactive programming concepts"
           - "Designs database models"
           - "Appropriately handles errors or failure conditions in their code"
+          - "Handles empty, error, and loading states gracefully in multi screen flows"
```

Comment:
How about calling out designers specifically? "Works with designers to handle empty..."